RoslynPluginGenerator/AnalyzerPluginGenerator.cs

```diff
@@ -97,8 +101,8 @@ private bool TryGenerateRulesFile(string packageDirectory, string nuGetDirectory
 
             this.logger.LogInfo(UIResources.APG_LocatingAnalyzers);
 
-            AnalyzerFinder finder = new AnalyzerFinder(this.logger);
-            IEnumerable<DiagnosticAnalyzer> analyzers = finder.FindAnalyzers(packageDirectory, nuGetDirectory);
+            DiagnosticAssemblyScanner diagnosticAssemblyScanner = new DiagnosticAssemblyScanner(this.logger, nuGetDirectory);
+            IEnumerable<DiagnosticAnalyzer> analyzers = diagnosticAssemblyScanner.InstantiateDiagnosticsFromDirectory(packageDirectory, language);
```

Comment:
*FindAnalyzers* is a better name, or just "InstantiateDiagnostics" - the first method parameter is called *directoryPath* so it's clear from the parameter name and doc comment that directories will be searched. There's no need to repeat "Directory" in the method name.